src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -81,6 +88,26 @@ export const FindPersonPanel = () => {
                   />
                 )}
               </Field>
+
+              <Field name="user.nvi.verifiedAt.id">
+                {({ field, form }: FieldProps<NviVerification>) => (
```

Comment:
dette stemmer vel ikke lenger nå som denne Fielden peker til selve id i stedet for hele objektet
```suggestion
                {({ field, form }: FieldProps<string>) => (
```